script/elevate-cpanel.PL

```diff
@@ -1138,7 +1138,14 @@ Restore removed packages during the previous stage.
 
 sub run_stage_4 ($self) {
     Cpanel::OS::flush_disk_caches();
-    Cpanel::OS::clear_cache_after_cloudlinux_update();    # I didn't pick the "clear cache but don't die" name...
+    Cpanel::OS::clear_cache_after_cloudlinux_update();
+
+    my $upgrade_errors = $self->leapp->check_upgrade_log_for_failures();
+    if ($upgrade_errors) {
+        my $message = sprintf("The LEAPP upgrade process did not succeed\n");
+        $message .= sprintf( "Review the log at %s for more details\n", Elevate::Constants::LOG_FILE );
+        die($message);
```

Comment:
QUESTION:  Since it is likely that leapp failed, do we want leapp to execute again when they run `--continue`?  If so, we need to bump the stage back down to 3 before we die or we will just jump right back into stage 4 and start with the post_leapp processes.

FIX:
* If we decide to leave it at 4, then the touch file message needs to make it clear to the customer that leapp will NOT execute again and that we expect the server to be upgraded before executing `--continue`
* If we decide to bump the stage back to 3, then the touch file message needs to make it clear to the customer that leapp will execute again when the execute `--continue`

Reply:
I would leave it at state 4 for now. The code immediately following this lowers it to 3 if needed.